tests/baseline/tsfresh_feature_names.py

```diff
@@ -0,0 +1,229 @@
+TSFRESH_BASELINE_VERSION = '0.3.0'
```

Comment:
It probably would be, but this is the methodology I used to write the test_tsfresh test for my application.  I found it useful on my application side, so I stopped dev on my application and thought I would port it and see if it was helpful for tsfresh.